Dockerfile

```diff
@@ -3,7 +3,7 @@
 # upgrade everything by simply updating the Alpine version.
 #
 # Alpine is updated every 6 months so all packages are pretty recent.
-FROM alpine:3.13
+FROM alpine:3.16
```

Comment:
The connection between Docker image and Chromium for CI is one of those things I have to keep relearning.